taurus_pyqtgraph/datainspectormodel.py

```diff
@@ -0,0 +1,205 @@
+
+                    point_x = curve_x_data[point_index][0]
+                    point_y = curve_y_data[point_index][0]
+                    try:
+                        # Painting the cloud with the values of
+                        # the selected data and blink the point base
+                        # on the i values which is the index of current curve
+                        self.paintCloud(point_x, point_y, i)
+                        self.triggerPoint(curve, point_index, i)
+
+                    except KeyError:
+
+                        # Create the TextItem object for current curve and init
+                        # the point style
+                        self.informationClouds[i] = TextItem()
+
+                        parent = self.plot.curves[i]
+                        self.informationClouds[i].setParentItem(parent)
+
+                        self.scatter[i] = curve.scatter
+                        if curve.opts['symbol'] is None:
+                            # create the custom style of the point
+                            # if point style (symbol and size) not exist
+                            curve.setSymbol('o')
+                            curve.setSymbolSize(10)
+                            self.custom_point_style_init = True
+
+                        self.paintCloud(point_x, point_y, i)
+                        self.triggerPoint(curve, point_index, i)
+
+                else:
+                    try:
+                        # Clean the information cloud
+                        # when the line isn't near any plot point
+                        self.informationClouds[i].setText("")
+                        curve.updateItems()
+                    except KeyError:
+                        pass
+
+    def paintCloud(self, x, y, i):
+        """
+        Method use to painting the cloud with the data
+
+        :param x: the new x position of the picked point
+        :param y: the new y position of the picked point
+        :param i: the index of current curve
+        """
+        self.informationClouds[i].setPos(x, y)
+        self.informationClouds[i].setHtml("<div style='%s'> "
+                                           "<span>x=%s "
+                                           "<span>y=%0.1f</span> "
+                                           "</div>" % (self.cloudStyle,
+                                                       self.timestampToTime(x),
+                                                       y))
+
+    def triggerPoint(self, curve, point_index, i):
+        """
+        Method use to trigger the point
+
+        :param curve: is the reference to the current curve
+        :param point_index: index of picked point
+        :param i: the index of current curve
+        """
+
+        # Update itemas to make sure the older point is not still trigger
+        curve.updateItems()
+        scatterPoint = self.scatter[i].points()[point_index[0]]
+        scatterPoint.setSize(20)
```

Comment:
instead of hardcoding, make this `20` a class attribute (so that it can be changed)

Reply:
I would not bother for now. We can la add them later on if we need them. (the idea is to only create the minimum necessary public API, so that we do not need to maintain it afterwards)